modules/virt-about-qemu-guest-agent-web.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-information.adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual Machine Overview* tab and the *Details* tab display information about the *Hostname*, *Operating system*, *Time zone*, and *Logged in users*.
+
+The *Virtual Machine Overview* shows information about the guest operating system installed on the virtual machine.
+
+The *Details* tab displays a table for *Logged in users.* This information includes *User name*, *Domain*, *Login time*, and *Elapsed Logged in time*.
```

Comment:
Same comment about reducing the number of actual GUI items shown.

```
 The *Details* tab displays a table for logged in users. This information includes the user name, domain, login time, and elapsed logged in time.
```

Though I might question if it's really necessary to call out all of the fields listed in the table.